src/components/dify-chatbot.module.css

```
@@ -0,0 +1,125 @@
+  }
+}
+
+:global(#dify-chatbot-bubble-window) {
+  width: 24rem !important;
+  height: 40rem !important;
+  position: fixed !important;
+  inset: auto 1rem 1rem auto !important;
+}
+
+@media (max-width: 640px) {
+  :global(#dify-chatbot-bubble-window) {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)